src/cascadia/TerminalSettingsEditor/AddProfile.xaml

```diff
@@ -68,21 +64,17 @@
                         </muxc:RadioButtons.ItemTemplate>
                     </muxc:RadioButtons>
                 </local:SettingContainer>
-                <Button x:Uid="AddProfile_DuplicateButton"
+                <Button x:Name="DuplicateButton"
                         Margin="{StaticResource StandardControlMargin}"
-                        AutomationProperties.AutomationId="AddProfile_DuplicateButton"
-                        AutomationProperties.Name="{Binding Tag, RelativeSource={RelativeSource Self}}"

```

Comment:
> `"{Binding Tag, RelativeSource={RelativeSource Self}}"`

wait is that some XAML magic for "use this other property on myself"? That's really neat!